packages/core/docs/guide/checkout.md

```diff
@@ -0,0 +1,239 @@
+# What's checkout?
+Checkout is a process of providing information about shipping and billing details, and picking certain shipping and payment method. Then with all the data, we are able to place an order and pay for it.
+
+## Shipping details
+Shipping details are information about the receiver and her/his address. Based on that, shop will know how to address a package.
+
+To access shipping details, we can use property returned by `useShipping` called `shipping`.
+```js{8,16}
+import { useShipping } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      load,
+      shipping
+    } = useShipping();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    return {
+      shipping
+    };
+  }
+}
+```
+`shipping` property will return `null` if the address is not saved on the server or has not been loaded.   
+
+In order to save shipping details on the server, you can use `save` method:
+```vue{2,15,24}
+<template>
+  <form @submit.prevent="save({ shippingDetails: shippingForm })">
+    <!-- form fields -->
+    <button type="submit" :disabled="loading">Submit</button>
+  </form>
+</template>
+
+<script>
+import { useShipping } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      save,
+      loading,
+      shipping
+    } = useShipping();
+
+    const shippingForm = ref(/* object for shipping details */);
+
+    return {
+      shippingForm,
+      save
+    };
+  }
+}
+</script>
+```
+
+Shipping details stored on the server with `save` method will be possible to load after a refresh with `load` method.
+
+## Shipping providers
+Shipping provider is an aggregator that provides us one or more shipping methods.    
```

Comment:
please mention that's integration, not only the "aggregator", always one provider components mean one 3rd party provider of either payments or shippings